.changeset/four-bugs-reply.md

```diff
@@ -0,0 +1,7 @@
+---
+"@salt-ds/core": minor
+---
+
+Updated `LinearProgress` to display a moving line to represent an unspecified wait time, when no `value` is set.
```

Comment:
```suggestion
Updated `LinearProgress` to display a moving line to represent an unspecified wait time, when no `value` is undefined.
```

? Because I'm getting value={undefined} will work